### PR TITLE
Feature/rtodling/gcm 1 12 4 tskinice realfix

### PR DESCRIPTION
**Pull request by rtodling**
These are fixes for TSKINICE - which so far are coming out as zero.

These require a change in HISTORY for consistency.

All instances of TSKINI in history are aliased to TSKINICE ... with this fix all instances should simply request TSKINICE and forget a aliasing. 

The changed HISTORY's are already committed, but in the GEOSgcm_App repo ... will try to see if no conflicts there and submit a PR.

### CODE REVIEW

**Review by gmao-rreichle — APPROVED** [review]
No impact on land, looks ok to me.

**Comment by sanAkel** [comment]
fixes https://github.com/GEOS-ESM/GEOSgcm_GridComp/issues/544

**Review by lcandre2 — APPROVED** [review]
Fine by me. No landice impact.

File: GEOSagcm_GridComp/GEOSphysics_GridComp/GEOSsurface_GridComp/GEOSsaltwater_GridComp/GEOS_CICE4ColumnPhysGridComp.F90 [hunk 1]
```diff
@@ -1329,9 +1329,9 @@ subroutine SetServices ( GC, RC )
   VERIFY_(STATUS)
 
   call MAPL_AddExportSpec(GC                    ,&
-    SHORT_NAME         = 'ISTSFC',                         &
+    SHORT_NAME         = 'TSKINICE',                  &
     LONG_NAME          = 'snow_or_ice_surface_temperature',&
-    UNITS              = 'C'                         ,&
+    UNITS              = 'K'                         ,&
```
Comment (sanAkel) [inline GEOSagcm_GridComp/GEOSphysics_GridComp/GEOSsurface_GridComp/GEOSsaltwater_GridComp/GEOS_CICE4ColumnPhysGridComp.F90:1334]: UNITS = 'C'

Comment (zhaobin74) [reply GEOSagcm_GridComp/GEOSphysics_GridComp/GEOSsurface_GridComp/GEOSsaltwater_GridComp/GEOS_CICE4ColumnPhysGridComp.F90:1334]: UNITS = 'K' is correct based on  @rtodling 's fix.

Comment (sanAkel) [reply GEOSagcm_GridComp/GEOSphysics_GridComp/GEOSsurface_GridComp/GEOSsaltwater_GridComp/GEOS_CICE4ColumnPhysGridComp.F90:1334]: well @zhaobin74 if the `result` is consistent with units, fine!

Screen Shot 2022-02-18 at 12 20 14 PM

Anyway that GC is _yours_ ...

**Review by sdrabenh — APPROVED** [review]

**Pull request merged**